packages/core/copy-uuid/src/components/CopyUuid.vue

```diff
@@ -130,9 +142,8 @@ const copyIdToClipboard = (executeCopy: (prop: string) => boolean) => {
   }
 
   const isTruncated = props.uuid.length > notifyTrimLength
-  const messagePrefix = props.isHidden
```

Comment:
question: did you intend to remove this conditional without replacing the logic?

Reply:
I think if `format` is `hidden` or `redacted` we don't want to show the uuid in the success message

Reply:
Fixed.